internal/operands/metrics/resources.go

```diff
@@ -16,6 +16,133 @@ const (
 	componentAlertLabelValue = "ssp-operator"
 )
 
+type MetricDesc struct {
+	Name    string
+	Expr    intstr.IntOrString
+	Summary string
+}
+
+var SspMetrics = func() []MetricDesc {
+	return []MetricDesc{
+		{
+			Name:    "kubevirt_ssp_operator_up_total",
+			Expr:    intstr.FromString("sum(up{pod=~'ssp-operator.*'}) OR on() vector(0)"),
+			Summary: "The total number of running ssp-operator pods",
+		},
+		{
+			Name:    "kubevirt_ssp_template_validator_up_total",
+			Expr:    intstr.FromString("sum(up{pod=~'virt-template-validator.*'}) OR on() vector(0)"),
+			Summary: "The total number of running virt-template-validator pods",
+		},
+		{
+			Name:    "kubevirt_ssp_num_of_operator_reconciling_properly",
+			Expr:    intstr.FromString("sum(ssp_operator_reconciling_properly)"),
+			Summary: "The total number of ssp-operator pods reconciling with no errors",
+		},
+		{
+			Name:    "kubevirt_ssp_rejected_vms_total",
+			Expr:    intstr.FromString("sum(increase(total_rejected_vms{pod=~'virt-template-validator.*'}[1h]))"),
+			Summary: "The total number of vms rejected by virt-template-validator",
+		},
+		{
+			Name:    "kubevirt_ssp_total_restored_common_templates",
+			Expr:    intstr.FromString("sum(increase(total_restored_common_templates{pod=~'ssp-operator.*'}[1h])) OR on() vector(0)"),
+			Summary: "The total number of common templates restored by the operator back to their original state",
+		},
+	}
+}()
+
+func getPrometheusRules() []promv1.Rule {
+	rules := []promv1.Rule{
```

Comment:
I think it would be nice to have consistent depth of reference, if we separated SspMetrics into a seperate var, lets separate Alerts rules as well.